app/views/organisation_member/requests/show.html.erb

```diff
@@ -44,7 +44,7 @@
         <dd><%= time_ago_in_words(@request.updated_at) %> ago</dd>
       </div>
       <div class="col-sm-6">
-        <% if @request.share.present? %>
+        <% if @request.share.present? && @request.share.unrevoked? %>
```

Comment:
how about putting this in Pundit, and instead called can :show, @request

Reply:
done in f029340